locales/en/app.ftl

```diff
@@ -216,4 +215,8 @@ copyright-info =
   Portions of this content are &#x24B8; 1998-2018 by individual mozilla.org contributors. <br />
   Content available under a  <a href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
 
-hibp-attribution = Breach data provided by
+# the following string contains HTML markup and should not be translated.
+hibp-link = <a id="hibp-link" href="https://www.haveibeenpwned.com" target="_blank" rel="noopener noreferrer">{-brand-HIBP}</a>
+
+# Breach data provided by Have I Been Pwned.
+hibp-attribution = Breach data provided by { hibp-link }.
```

Comment:
@flodolo @pdehaan  Not sure if it would perhaps be easier on translators to handle the hibp attribution string and link like this.

Reply:
Can we make this

```
 hibp-attribution = Breach data provided by { $hibp-link }. 
```

and then deal with `$hibp-link` in the code? Right now, that string is exposed to localizers but it doesn't have anything localizable (actually, you're adding the risk of people breaking the markup).

Reply:
Yep, this will work.